README.md

```diff
@@ -18,6 +18,9 @@ INSTALLATION
 
 1. Upload all files of this folder (IMPORTANT: also upload the invisible .htaccess files)
 2. Make the directories data/cache, data/favicons, data/logs, data/thumbnails, data/sqlite and public/ writeable
+
+   `chmod o+w selfoss/data/cache selfoss/data/favicons selfoss/data/logs selfoss/data/thumbnails selfoss/data/sqlite selfoss/public/`
```

Comment:
Many users will be uploading selfoss to shared hosts where they do not have terminal access. I think I would rather keep the readme simple and explain it in detail in a separate installation guide.